src/components/Projects/index.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function index() {
```

Comment:
Semantics, i know, but i am a stickler for every detail 😄 this should be `Project` instead of `index` for the component name + export